src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+.body {
+  margin: 0;
+  padding: 0 8px;
+}
+
+.form {
+  margin-top: 20px;
+  display: flex;
+}
+
+.search_bar {
```

Comment:
your tests are failing because your inputs are higher than they should be, on a design it's 70px and yours is 74
![Знімок екрана 2023-07-31 о 11 46 19](https://github.com/mate-academy/layout_search-bar-airbnb/assets/106310796/90d03729-99e3-4481-a435-3b0ffbb362ef)
<img width="363" alt="Знімок екрана 2023-07-31 о 11 46 24" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/106310796/01d5ef98-ae0a-4de7-8411-c7d578d7219c">